installer/scripts/service_control

```diff
@@ -215,7 +227,9 @@ enable_omsagent_service()
         fi
         touch  /etc/opt/microsoft/omsagent/conf/.service_registered
         start_omsagent
-    fi
+    else
+        exit_if_agent_not_onboarded
```

Comment:
We know we're not onboarded at this point, yes? Thus, no need to do the 'if' test again. Not a biggie, just a nit.